zwave_me_server/rootfs/etc/cont-init.d/z-way-server-init

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/with-contenv bashio
+
+bashio::log.info "Preparing Z-Way server to start..."
+
+# Default config path
+defCJ=/opt/z-way-server/automation/defaultConfigs/config.json
+
+bashio::log.info "Checking and updating default config path..."
+
+# Change device path in /defailtConfig/config.json
+if [ -f "$defCJ" ]; then
+    sed -Ei "s|/dev/tty[A-Z]*[0-9]|$(bashio::config 'device')|g" $defCJ
```

Comment:
Better to us jq to access the right app. Otherwise you can't use two ports (Z-Wave + Zigbee)